cmd/restic/cmd_forget_test.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	rtest "github.com/restic/restic/internal/test"
+)
+
+func TestPreventNegativeForgetOptionValues(t *testing.T) {
+	invalidForgetOpts := []ForgetOptions{
+		{Last: -2},
+		{Hourly: -2},
+		{Daily: -2},
+		{Weekly: -2},
+		{Monthly: -2},
+		{Yearly: -2},
```

Comment:
Please also add a basic test to check that -1 and positive counts are accepted.

Reply:
Added tests for -1, 0, and positive values, as well as tests for valid and invalid `Within*` options.